website/versioned_docs/version-v1.x/circuits.md

```diff
@@ -7,45 +7,407 @@ sidebar_position: 7
 
 # Circuits
 
-MACI has three zk-SNARK circuits:
+MACI has three main zk-SNARK [circuits](https://github.com/privacy-scaling-explorations/maci/tree/dev/circuits):
 
-1. `ProcessMessages.circom`, which takes a batch of messages, and updates the
-   state and ballot trees according to the contents of said messages.
-2. `TallyVotes.circom`, which counts votes from users' ballots, batch by batch.
-3. `Subsidy.circom`, which implements [pairwise subsidy](https://hackmd.io/@chaosma/H1_9xmT2K)
+1. [`ProcessMessages.circom`](https://github.com/privacy-scaling-explorations/maci/blob/dev/circuits/circom/processMessages.circom), which takes a batch of encrypted messages, decrypts them, and generates a proof that the coordinator's local processing was performed correctly.
```

Comment:
Nitpick but I think this should be `ProcessMessages` (circuit name, PascalCase) or `processMessages.circom` (file name, camelCase)

Naming convention question: _should_ Circom files be camelCase?

Reply:
Might just be easiest to update the filenames to be PascalCase to match the circuit names?

Reply:
Not sure about conventions, here is how [circomlib](https://github.com/iden3/circomlib/tree/d5ed1c3ce4ca137a6b3ca48bec4ac12c1b38957a/circuits) names them (all lower it seems)